Advanced_Projects/Audio Controlled File Navigation System/file_navigation.py

```
@@ -0,0 +1,146 @@
+            text_dict = json.loads(text)
+            recognized_text = text_dict['text'].lower().strip()
+            print(f"Recognized: {recognized_text}")
+
+            if 'list' in recognized_text:
+                self.list_dir()
+                
+            elif 'up' in recognized_text:
+                self.navigate_up()
+
+            elif "down" in recognized_text:
+                folder = recognized_text.split()[-1];
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)